src/test/instance.lua

```diff
@@ -16,7 +16,12 @@ box.once('init', function()
     box.schema.user.create('test_ordin', { password = '2HWRXHfa' })
     box.schema.user.create('test_admin', { password = '4pWBZmLEgkmKK5WP' })
 
-    box.schema.user.grant('test_ordin', 'read,write', 'user')
-    box.schema.user.grant('test_admin', 'execute',    'super')
+    box.schema.user.grant('test_ordin', 'read,write', 'space', 'user')
+    box.schema.user.grant('test_admin', 'super')
 end)
 
+-- Java has no internal support for unix domain sockets,
+-- so we will use tcp for console communication.
+console = require('console')
+console.listen(3313)
+console.start()
```

Comment:
Removed the last line. But kept the others because the tests are faster with tcp console.
So we have now a choice of tcp console and tarantoolctl console (enter instance) in tests.